tests/Integration/Behaviour/Features/Context/Domain/OrderPaymentFeatureContext.php

```diff
@@ -52,13 +52,17 @@ public function addPaymentToOrderWithTheFollowingDetails(string $orderReference,
 
         $data = $table->getRowsHash();
 
+        $currencyId = is_numeric($data['id_currency']) ?
```

Comment:
I don't think we should use IDs here to fetch the currency, as ID are not set in stone and can change depending on how we will build the fixtures in the future But it's not to be fixed now anyway, let's see this matter in the PR that will fix the tests

Reply:
fixed ;)